src/main/java/org/broadinstitute/consent/http/service/UserService.java

```diff
@@ -201,6 +201,10 @@ public void updateEmailPreference(boolean preference, Integer userId) {
         return users.stream().map(SimplifiedUser::new).collect(Collectors.toList());
     }
 
+    public void deleteUserRole(Integer dacUserId, Integer roleId) {
+        userRoleDAO.removeSingleUserRole(dacUserId, roleId);
+    }
```

Comment:
Let's add some logging here, this is something we need to do more of. Let the method take a User as the user doing the action, and log (INFO) something like:
```
User 'Gregory Rushton' deleting Role Id: roleId from User ID: dacUserId
```